python/modem-firmware-1.3+/create_device_credentials.py

```diff
@@ -183,6 +194,13 @@ def main():
     if (len(args.csr) == 0):
         priv = OpenSSL.crypto.dump_privatekey(OpenSSL.crypto.FILETYPE_PEM, priv_key)
         write_file(args.path, args.fileprefix + common_name + "_prv.pem", priv)
+        if args.embed_save:
+            write_file(args.path, "private-key.pem", embed_save_convert(priv))
+
+    if args.embed_save:
+        # save the AWS CA cert
+        write_file(args.path, "ca-cert.pem",
+                   embed_save_convert(device_credentials_installer.aws_ca.encode('utf-8')))
```

Comment:
Maybe the aws_ca and other useful data should be in a separate module rather than import from device_credentials_installer?

Reply:
yeah i was thinking that too, but just wanted to keep it simple and modify one file...
zou discovered that this change breaks `device_credentials_installer.py` 😓 
so, i'll do this now.